sktime/forecasting/base/_fh.py

```diff
@@ -149,7 +155,11 @@ def __new__(cls, values=None, is_relative=None):
             setattr(cls, method, _delegator(method))
         return object.__new__(cls)
 
-    def __init__(self, values=None, is_relative=True):
+    def __init__(
+        self,
+        values: Optional[Union[VALID_FORECASTING_HORIZON_TYPES]] = None,
```

Comment:
I think you don´t need the `Optional` if the default is already `None` (then `Optional` is inferred)?

Reply:
you are right. I have reread one more time https://docs.python.org/3/library/typing.html#typing.Optional and removed `Optional`